test/e2e/inspect_test.go

```diff
@@ -46,7 +46,7 @@ var _ = Describe("Podman inspect", func() {
 		SkipIfRemote()
 		session := podmanTest.Podman([]string{"inspect", "foobar4321"})
 		session.WaitWithDefaultTimeout()
-		Expect(session.ExitCode()).To(Not(Equal(0)))
+		Expect(session.ExitCode()).To(Not(Equal(1)))
```

Comment:
This would pass on a non-error exit. Can we explicitly specify the exit code here?